PixelHunter1995/Player.cs

```diff
@@ -57,20 +57,21 @@ public void Update(GameTime gameTime)
             if (game.IsActive && mouseState.LeftButton == ButtonState.Pressed)
             {
                 this.MovePosition = new Vector2(mouseState.X, mouseState.Y);
+                Say("Hi, I'm the player!");  // For debugging porpoises
             }
             this.MoveDirection = MovePosition - Position;
             this.Position = this.Approach(Position, MovePosition, 2);
+            CharComp.Update(gameTime);
```

Comment:
shouldn't player have update function that forward to CharComp like for loadContent?